aws/s3.tf

```diff
@@ -123,3 +123,94 @@ module "s3_bucket" {
 
   tags = local.tags
 }
+# aws S3 backup //////////////////////
+resource "aws_backup_vault" "s3_backup_vault" {
+  name = "s3-backup-vault-${var.name}-${var.environment}"
```

Comment:
A prefix is better than a suffix in this case.

Reply:
ok! rewrited!